test/parallel/parallel.status

```diff
@@ -11,6 +11,8 @@ prefix parallel
 test-timers-immediate-queue: PASS,FLAKY
 # https://github.com/nodejs/node/issues/23277
 test-worker-memory: PASS,FLAKY
+# https://github.com/nodejs/node/issues/41206
+test-crypto-keygen: SLOW
```

Comment:
Marking a test `SLOW` has no effect unless the test runner is run with `--flaky-tests=skip` which is not the case for Windows test builds (which run with `--flaky-tests=dontcare` but `SLOW` test are not `FLAKY`): https://github.com/nodejs/node/blob/a2572940cdff72cdaa361b78389f05e5b91de315/tools/test.py#L1703-L1704

See https://github.com/nodejs/node/pull/23251 for when `SLOW` was introduced.

Reply:
@richardlau thanks for pointing that out. Updated to mark as flaky instead.